From f03acf0a9c17435b6ac3f3c9dc8a8c6615c629a1 Mon Sep 17 00:00:00 2001
From: Sandeep Patil <sspatil@google.com>
Date: Wed, 9 Oct 2019 21:38:04 -0700
Subject: [PATCH 22/50] ANDROID: staging: ion: fix sparse warning in ion system
 heap

drivers/staging/android/ion/heaps/ion_system_heap.c:250:24: sparse: sparse:
symbol 'system_heap' was not declared. Should it be static?

Fixes: 142434553
Test: Build and boot cuttlefish

Change-Id: I6e1eb9365ca12c2d081533ed2db9881086531ec7
Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Sandeep Patil <sspatil@google.com>
---
 drivers/staging/android/ion/heaps/ion_system_heap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/android/ion/heaps/ion_system_heap.c b/drivers/staging/android/ion/heaps/ion_system_heap.c
index 927ab439d..5eb8ea6c9 100644
--- a/drivers/staging/android/ion/heaps/ion_system_heap.c
+++ b/drivers/staging/android/ion/heaps/ion_system_heap.c
@@ -244,7 +244,7 @@ static struct ion_heap_ops system_heap_ops = {
 	.shrink = ion_system_heap_shrink,
 };
 
-struct ion_system_heap system_heap = {
+static struct ion_system_heap system_heap = {
 	.heap = {
 		.ops = &system_heap_ops,
 		.type = ION_HEAP_TYPE_SYSTEM,
-- 
2.17.1

